### DISCUSSION TITLE
DHCP  limitation?

### DISCUSSION THREAD

**Discussion by tkirkland** [BeryJu/gravity#1180] [Q&A]
Does the DHCP only support these?

![image](https://github.com/user-attachments/assets/6efdbc45-97ac-49d7-a6c5-dc440a13062b)

**Comment by adfrancis** [comment]
No, it seems you can add raw tags for any DHCP scope option you want. The options you posted a screenshot of are just the "named tags" (i.e easy to read DHCP tags).

**Reply by tkirkland** [reply]
Apologies.  Why my inquiry was meant to convert was, 'Does the app support all of the standard options (100+) or only those listed?'

If I understand your reply correctly, I can use any option, though some might be missing the tag and have to be used by number.

**Comment by adfrancis** [comment]
Yes thats my understanding. I have configured a few DHCP options via a "tag" number as opposed to the "tagName" syntax

**Comment by tkirkland** [comment]
Perfect. That's what I was asking about. I don't mind using numbers, it was
kind of unclear though. Thank you.
